### PR TITLE
Fix PlanPrinter formatting

### PR DESCRIPTION
**Pull request by akshatnair**
Ensure consistent printing for rows as long in various components.
Do not print memory component of cost in bytes.

### CODE REVIEW

**Comment by akshatnair** [comment]
New formatted output is

`
ScanFilter[table = tpch:tpch:orders:sf10.0, originalConstraint = ("custkey" > BIGINT '10'), filterPredicate = ("custkey" > BIGINT '10')] => [custkey:bigint]                    
                     Cost: {rows: 15000000, bytes: 143.05MB, cpu: 150000000.00, memory: 0.00, network: 0.00}/{rows: 14999910, bytes: 143.05MB, cpu: 300000000.00, memory: 0.00, network: 0.00} 
                     CPU fraction: 78.36%, Input: 15000000 rows (0B), Output: 14999895 rows (128.75MB), Filtered: 0.00%                                                                        
                     Input avg.: 3750000.00 lines, Input std.dev.: 0.00%                                            
`

**Review by rschlussel-zz — APPROVED** [review]

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java [hunk 1]
```diff
@@ -1325,19 +1334,14 @@ private String formatPlanNodeStatsAndCost(PlanNode node)
...
+                    formatDouble(cost.getMemoryCost()),
+                    formatDouble(cost.getNetworkCost()));
         }
     }
 
-    private static String formatEstimate(double value)
-    {
-        return isNaN(value) ? "?" : String.valueOf(value);
-    }
-
```
Comment (losipiuk) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java:1340]: `formatEstimateAsDataSize` is probably not used any more, drop it too.

**Review by losipiuk — COMMENTED** [review]
One comment.
Make sure you backport the change to `epic/statistics-12` branch too.

**Review by kbajda — COMMENTED** [review]
@akshatnair LGTM but maybe we can unify a way to display sizing too...

Now we have:
in Cost :  rows: 15000000, bytes: 143.05MB
in CPU fraction:  Output: 14999895 rows (128.75MB)

how about having: rows: 15000000 (143.05MB)

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java (hunk 1 above)
Comment (akshatnair) [reply presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java:1340]: Nope, it is still used for "bytes: " formatEstimateAsDataSize(stats.getOutputSizeInBytes())

**Review by losipiuk — APPROVED** [review]

**Comment by akshatnair** [comment]
cherry-picked on release and statistics-12 branches.

**Pull request closed without merging**